lusSTR/filter_settings.py

```diff
@@ -98,181 +102,188 @@ def thresholds(filter, metadata, locus_reads, quest_al_reads):
-        all_thresh = (
-            np.ceil(stutter_thresh * al1_ref_reads) +
-            forward_stut_thresh(forward_thresh, stutter_thresh, ref_reads)
+    if all_type == "-1_stutter":
+        all_thresh = np.ceil(stutter_thresh * al1_ref_reads) + forward_stut_thresh(
+            forward_thresh, stutter_thresh, ref_reads
         )
-        all_type = output_allele_call(
-            quest_al_reads, all_thresh, '+1_stutter/-2_stutter'
+        all_type = output_allele_call(quest_al_reads, all_thresh, "-1_stutter/+1_stutter")
+    elif all_type == "-2_stutter":
+        all_thresh = np.ceil(stutter_thresh * al1_ref_reads) + forward_stut_thresh(
+            forward_thresh, stutter_thresh, ref_reads
         )
+        all_type = output_allele_call(quest_al_reads, all_thresh, "+1_stutter/-2_stutter")
     elif quest_al_reads <= forward_stut_thresh(forward_thresh, stutter_thresh, ref_reads):
-        all_type = '+1_stutter'
-        stut_perc = round(quest_al_reads/ref_reads, 3)
+        all_type = "+1_stutter"
+        stut_perc = round(quest_al_reads / ref_reads, 3)
     else:
-        all_type = 'real_allele'
+        all_type = "real_allele"
     return all_type, stut_perc
 
 
 def allele_type_ru(
     ref, ru, all_type, metadata, quest_al_reads, ref_reads, al1_ref_reads, all_type_df
 ):
-    stutter_thresh = metadata['StutterThresholdDynamicPercent']
-    forward_thresh = metadata['StutterForwardThresholdDynamicPercent']
+    stutter_thresh = metadata["StutterThresholdDynamicPercent"]
+    forward_thresh = metadata["StutterForwardThresholdDynamicPercent"]
     stutter_thresh_reads = np.ceil(stutter_thresh * ref_reads)
     stut_perc = None
     allele_diff = round(ref - ru, 1)
     if allele_diff == 1 and ref_reads > quest_al_reads:  # -1 stutter
         all_type, stut_perc = minus1_stutter(
-            all_type, stutter_thresh, forward_thresh, stutter_thresh_reads, ref_reads,
-            al1_ref_reads, quest_al_reads
+            all_type,
+            stutter_thresh,
+            forward_thresh,
+            stutter_thresh_reads,
+            ref_reads,
+            al1_ref_reads,
+            quest_al_reads,
         )
     elif allele_diff == 2 and ref_reads > quest_al_reads:  # -2 stutter
-        if check_2stutter(all_type_df, 'ru', ru)[0] is True:
-            ref_reads = check_2stutter(all_type_df, 'ru', ru)[1]
+        if check_2stutter(all_type_df, "ru", ru)[0] is True:
+            ref_reads = check_2stutter(all_type_df, "ru", ru)[1]
             stutter_thresh_reads = stutter_thresh * ref_reads
             all_type, stut_perc = minus2_stutter(
-                all_type, stutter_thresh_reads, forward_thresh, stutter_thresh_reads,
-                ref_reads, al1_ref_reads, quest_al_reads
+                all_type,
+                stutter_thresh_reads,
+                forward_thresh,
+                stutter_thresh_reads,
+                ref_reads,
+                al1_ref_reads,
+                quest_al_reads,
```

Comment:
And then about seventy bajillion changes related to quotation marks and long parameter lists.